### PR TITLE
Initial start of splitting into user and development

### PR DESCRIPTION
**Pull request by j1nx**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -4,67 +4,70 @@ nav:
...
+            - Plugin Manager: OPM.md
+            - Speech To Text: stt_plugins.md
+            - Text To Speech: tts_plugins.md
+            - Wake Word: ww_plugins.md
+            - Voice Activity Detection: vad_plugins.md
+            - Grapheme to Phoneme: g2p_plugins.md
+            - Audio Playback: audio_plugins.md
+            - Language Detection/Translation: lang_plugins.md
+            - Platform/Hardware Abstraction Layer: PHAL.md
+        - Technologies:
```
Comment (JarbasAl) [inline mkdocs.yml:56]: this section should go under user

File: mkdocs.yml [hunk 2]
```diff
@@ -4,67 +4,70 @@ nav:
...
+            - Common Play: OCP.md
+            - Personal Backend: personal_backend.md
+            - Skills Manager: osm.md
+            - Config Assistant: OCA.md
+            - STT Server: stt_server.md
+            - TTS Server: tts_server.md
+            - Translate Server: translate_server.md
+        - Mycroft:
```
Comment (JarbasAl) [inline mkdocs.yml:64]: this section should go under user

File: mkdocs.yml [hunk 3]
```diff
@@ -4,67 +4,70 @@ nav:
...
+            - Dinkum: dinkum.md
+            - Compatibility: compat.md
+            - OVOS vs MyCroft: mycroft.md
+        - Images:
```
Comment (JarbasAl) [inline mkdocs.yml:68]: this section should go under user

File: mkdocs.yml [hunk 4]
```diff
@@ -4,67 +4,70 @@ nav:
...
-        - Getting Started: setup_getting_started.md
         - License: license.md
-    - Overview:
+        - Friends: friends.md
         - F.A.Q.: faq.md
+    - User Guide:
+        - Getting Started:
+            - Downloading an image: setup_getting_started.md
+    - Development Guide:
         - Backend Services: backend.md
```
Comment (JarbasAl) [inline mkdocs.yml:14]: this backend intro fits in both sections, perhaps it should be split at some point once more info is added that warrants 2 pages

File: mkdocs.yml (hunk 1 above)
Comment (j1nx) [reply mkdocs.yml:75]: It is to technical for the user guide, but added an advanced section to it so will move it there.

STT / TTS for the normal user guide from the setup skill, the above in the advanced section.

agree?

File: mkdocs.yml (hunk 2 above)
Comment (j1nx) [reply mkdocs.yml:83]: See above🤦‍♂️

File: mkdocs.yml (hunk 3 above)
Comment (j1nx) [reply mkdocs.yml:87]: Indeed, not yet sure where this belongs

File: mkdocs.yml (hunk 4 above)
Comment (j1nx) [reply mkdocs.yml:33]: On it

**Pull request merged**